components/profile/pages/voting-power/sub-title/view.tsx

```diff
@@ -0,0 +1,79 @@
+
+import TokenSymbolView from "components/common/token-symbol/view";
+import InfoTooltip from "components/info-tooltip";
+import ResponsiveWrapper from "components/responsive-wrapper";
+
+import { formatStringToCurrency } from "helpers/formatNumber";
+
+interface ResponsiveProps {
+  xs: boolean;
+  md: boolean;
+}
+
+export default function VotingPowerSubTitleView({
+  label,
+  total,
+  votesSymbol,
+  propsMobile,
+  propsDesktopAndTablet,
+  infoTooltip,
+  getTextColorProps,
+  getTitleSpanClass,
+  getAmountClass,
+}: {
+  label: string;
+  infoTooltip: string;
+  total: string;
+  votesSymbol: string;
+  propsMobile: ResponsiveProps;
+  propsDesktopAndTablet: ResponsiveProps;
+  getTextColorProps: () => { className: string } | { style: { color: string } };
+  getTitleSpanClass: (type: string) => string;
+  getAmountClass: (type: string) => string;
+}) {
+
+  function handleAmount(icon = true) {
```

Comment:
`handle` makes it sound like this function makes _an action_ while in truth it's a _render_ function, rename it please